src/screens/ProfileScreen/index.js

```diff
@@ -212,13 +213,13 @@
   const {interactionsComplete} = useAfterInteractions();
   const isNotFromHomeTab = route?.params?.isNotFromHomeTab;
   const [, setIsHitApiFirstTime] = React.useState(false);
+  const [initialFollowerData, setInitialFollowerData] = React.useState([]);
 
   const updateUserClient = useUpdateClientGetstreamHook();
   const {refreshCount} = useResetContext();
   const {mappingColorFeed} = useCoreFeed();
   const LIMIT_PROFILE_FEED = 10;
   const TYPE_GALLERY = 'gallery';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new state variable `initialFollowerData` is introduced and initialized with an empty array. This seems to be used for storing the initial follower data fetched from the API.
```diff
+  const [initialFollowerData, setInitialFollowerData] = React.useState([]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->